### PR TITLE
Update of configuration.md 

### PR DESCRIPTION
**Pull request by MarcusE1W**
updated link to codemirror and enhanced the description how to use themes

### CODE REVIEW

**Comment by MarcusE1W** [comment]
This is a small test request. Let me know if the approach is ok.

**Comment by sbauer322** [comment]
I'll be able to check this out over the weekend, sorry for the delay on this and the other PRs you opened!

File: configuration.md [hunk 1]
```diff
@@ -17,19 +17,33 @@ For example, to turn on the line number gutter, find the `:editor` tag and in th
...
+> [:editor :lt.objs.style/set-theme  "rubyblue" ]
 
-To try third party themes, like [these ones](https://github.com/FarhadG/code-mirror-themes), download them and add an entry for it in user.behaviors file:  `[:app :lt.objs.style/provide-theme "theme-name" "/full/path/to/theme/css"]`. Once a theme is provided you can set it with the aforementioned `set-theme`.
+To see what different themes look like, try [the CodeMirror theme demo](http://codemirror.net/demo/theme.html).
+
+Note: Some themes listed there may not be available because Light Table may not be on the same CodeMirror version as the demo.
+
+**Third party themes**
+
+You have two options, many third party theme packages can be found in the plugin manager.
```
Comment (sbauer322) [inline configuration.md]: Could you clarify what the two options are? This line seems out of place.

File: configuration.md [hunk 2]
```diff
@@ -17,19 +17,33 @@ For example, to turn on the line number gutter, find the `:editor` tag and in th
...
+
+Or to try other third party themes, like [these ones](https://github.com/FarhadG/code-mirror-themes),
+1. download a third party theme
+2. check if the local LightTable user themes folder  .config/LightTable/User/core/css/themes/ already exists
```
Comment (sbauer322) [inline configuration.md]: Wrap .config/LightTable/User/core/css/themes/ with \`... so:

`.config/LightTable/User/core/css/themes/`

File: configuration.md [hunk 3]
```diff
@@ -2,12 +2,12 @@
...
 There are four main areas allowing for configuration within Light Table:
 
-* Settings \(aka Behaviors\)
+* Settings (aka Behaviors)
 * Keybindings
 * Themes
 * User Plugin
 
-### Change settings \(e.g., fonts, line numbers\)?
+### Change settings (e.g., fonts, line numbers)?
```
Comment (sbauer322) [inline configuration.md]: The '?' can be removed from this heading and in all the other headings below.

**Review by sbauer322 — CHANGES_REQUESTED** [review]
Just a few minor changes and this looks good to me! Thank you!

File: configuration.md (hunk 3 above)
Comment (MarcusE1W) [reply configuration.md]: done, all '?' removed

File: configuration.md (hunk 1 above)
Comment (MarcusE1W) [reply configuration.md]: I have changed the sentence a bit, without dwelling on the options...

File: configuration.md (hunk 2 above)
Comment (MarcusE1W) [reply configuration.md]: done

**Comment by MarcusE1W** [comment]
So, the changes are done 

- [x] ? removed

- [x] Two options... sentence rephrased

- [x] .config/... marked as code

**Comment by MarcusE1W** [comment]
ups, I closed the request accidentially for a moment. open again

**Comment by MarcusE1W** [comment]
Do you think this could be reviewed again ? Or waht should be the next step ?

**Comment by MarcusE1W** [comment]
Seems not to happen

**Pull request closed without merging**